### PR TITLE
Landing

### PR DESCRIPTION
**Pull request by heatherbooker**

### CODE REVIEW

**Comment by heatherbooker** [comment]
@dharness

sorry for bootstrapcrap

File: public/javascripts/submit.js [hunk 1]
```diff
@@ -2,29 +2,7 @@ $(document).ready(function() {
```
Comment (dharness) [inline public/javascripts/submit.js]: please remove

File: public/libs/bootstrap/config.json [hunk 2]
```diff
@@ -23,11 +139,256 @@
...
+    "forms.less",
+    "responsive-utilities.less",
+    "navs.less",
+    "navbar.less",
+    "responsive-embed.less"
   ],
   "js": [],
-  "customizerUrl": "http://getbootstrap.com/customize/?id=d80379b179c05d269f571e9ddf76743e"
+  "customizerUrl": "http://getbootstrap.com/customize/?id=f54fa87cc26d0d9ffc37be9ac18caaaa"
 }
```
Comment (dharness) [inline public/libs/bootstrap/config.json]: This file is disgusting -  is it absolutely necessery ?

File: public/libs/bootstrap/css/bootstrap.css [hunk 3]
```diff
@@ -5,8 +5,8 @@
  */
```
Comment (dharness) [inline public/libs/bootstrap/css/bootstrap.css:5]: why did you not use a CDN ?

File: public/stylesheets/landing.css [hunk 4]
```diff
@@ -0,0 +1,43 @@
+canvas {
```
Comment (dharness) [inline public/stylesheets/landing.css]: please do not style elements directly unless absolutely necessary. Give this a class name.

File: public/stylesheets/landing.css [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+  position: absolute;
+}
+
+.ccbtnStart {
```
Comment (dharness) [inline public/stylesheets/landing.css]: please use a dash before a prefix: cc-btn-strart

File: public/stylesheets/style.css [hunk 6]
```diff
@@ -1,13 +1,45 @@
...
 
 a {
-  color: #00B7FF;
+  color: inherit;
+}
+a:hover {
+  text-decoration: none;
+}
+
+.ccnavbar {
```
Comment (dharness) [inline public/stylesheets/style.css]: I don't think this belongs in this file
What is this file?
Is it the base styles? If so, call it base.css

File: public/stylesheets/style.css [hunk 7]
```diff
@@ -1,13 +1,45 @@
...
+  border-bottom: 2px solid #F0F0F0;
+}
+
+.cclogo {
```
Comment (dharness) [inline public/stylesheets/style.css]: same as above

File: public/stylesheets/style.css [hunk 8]
```diff
@@ -1,13 +1,45 @@
...
+  font-family: 'Pacifico', cursive;
+  display: inline-block;
+  margin: 20px 5px;
+  font-size: 3.3rem;
 }
 
-.chat {
-  height: 400px;
-  overflow-y: auto;
+.cclogin {
```
Comment (dharness) [inline public/stylesheets/style.css]: same as above

File: public/stylesheets/style.css [hunk 9]
```diff
@@ -1,13 +1,45 @@
...
+  float: right;
+  display: inline-block;
+  padding: 10px;
+  font-size: 2rem;
 }
+.cclogin:hover {
```
Comment (dharness) [inline public/stylesheets/style.css]: same as above

File: public/stylesheets/style.css [hunk 10]
```diff
@@ -1,13 +1,45 @@
...
+  cursor: pointer;
+  font-weight: bold;
+}
+
+@media (max-width: 360px) {
```
Comment (dharness) [inline public/stylesheets/style.css]: same as above

File: views/index.pug [hunk 11]
```diff
@@ -1,17 +1,20 @@
...
-      input(type='text' id='messageInput')
-      button(type='submit' id='submit') Submit
-    div.col-md-4.chat
-      h3 Incoming messages:
-      p.incoming
-    div.col-md-4.chat
-      h3 Outgoing messages:
-      p.outgoing
-
+  div.container-fluid
```
Comment (dharness) [inline views/index.pug]: `.container-fluid` is equivalent to `div.container-fluid`.
Prefer the former.

File: public/stylesheets/landing.css (hunk 4 above)
Comment (heatherbooker) [inline public/stylesheets/landing.css]: it actually is necessary to target like this in order to override bootstrap having done the same

Comment (dharness) [inline public/stylesheets/landing.css]: if you are **more** specific than bootstrap, your style will override it. Try something like
`canvas.floating-dots {}`

Comment (heatherbooker) [inline public/stylesheets/landing.css]: i did

File: views/index.pug (hunk 11 above)
Comment (heatherbooker) [inline views/index.pug]: only for container fluid?

**Comment by dharness** [comment]
LGTM

**Pull request merged**